.github/workflows/stylecheck-prs.yml

```diff
@@ -0,0 +1,25 @@
+name: stylecheck
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  stylecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: get changed Files
+        id: get_changed_files
+        uses: lots0logs/gh-action-get-changed-files@2.1.4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: run vale
+        uses: errata-ai/vale-action@master
+        with:
+          styles: https://github.com/errata-ai/Google/releases/latest/download/Google.zip
+          config: https://raw.githubusercontent.com/blockstack/docs/master/.vale/vale.ini
+          files: "${{ steps.get_changed_files.outputs.all }}"
```

Comment:
Is there any way we can exclude some files from being analyzed? For example, vale should never analyze files changes in `.github`. Maybe it would be easier to just specify certain dirs like `src` and `public` (and any others)?

Reply:
good point... uhm let me see what is possible here

Reply:
looks like this might be an option: https://docs.errata.ai/vale/config#format-associations. would still analyze the files but ignore them it looks like